src/frontend-common/common_host_interface.cpp

```diff
@@ -3344,6 +3353,18 @@ void CommonHostInterface::ReloadPostProcessingShaders()
     AddOSDMessage(TranslateStdString("OSDMessage", "Post-processing shaders reloaded."), 10.0f);
 }
 
+void CommonHostInterface::ToggleWidescreen()
+{
+  g_settings.gpu_widescreen_hack = !g_settings.gpu_widescreen_hack;
```

Comment:
Would it be better to switch between the chosen AR and native here, rather than 16:9?

Reply:
You're right, of course.  I've pushed another commit so that it does this.